### DISCUSSION TITLE
Image bit depth

### DISCUSSION THREAD

**Discussion by Justin-Krieg** [MathOnco/valis#119] [Q&A]
Hi @cdgatenbee 
Just a quick question, I was considering changing the bit rate on my slide acquisition from 16 to 14 bits, we just don't need that much information stored. Are there any broader implications for VALIS if I were to do this? I understand VALIS outputs in 16 bit, so there would be some transformation to those pixels, has that been implemented? The main points for doing this is to reduce initial acquisition file sizes, and also it can be quite difficult titrating minimum and maximum values in qupath on that scale, particularly after VALIS normalises the files. I will optimise these scans and try VALIS in the interim.

**Comment by cdgatenbee** [comment]
Hi @Justin-Krieg,
Unfortunately, I don't have any 14bit images, so I can't say for sure what the implications would be. However, Valis mainly uses libvips and bio-formats to convert and save the images, and based on my experience with those I don't recall 14bit being one of the supported datatypes. Still, it might be worth creating a pair of 14bit images to test valis with. If you end up going that route, please let me know how it goes.

Best,
-Chandler
